app/src/main/java/org/simple/clinic/instantsearch/InstantSearchResultsItemType.kt

```diff
@@ -19,29 +20,35 @@ sealed class InstantSearchResultsItemType : ItemAdapter.Item<InstantSearchResult
 
     fun from(
         patientSearchResults: List<PatientSearchResult>,
-        currentFacility: Facility
+        currentFacility: Facility,
+        searchQuery: String
     ): List<InstantSearchResultsItemType> {
       val (assignedFacilityPatients, nearbyFacilitiesPatients) = patientSearchResults
           .partition { it.assignedFacilityId == currentFacility.uuid }
 
-      val assignedFacilityPatientsGroup = generateAssignedFacilityPatientsGroup(assignedFacilityPatients, currentFacility)
-      val nearbyFacilitiesPatientsGroup = generateNearbyFacilitiesPatientsGroup(nearbyFacilitiesPatients, currentFacility)
+      val assignedFacilityPatientsGroup = generateAssignedFacilityPatientsGroup(assignedFacilityPatients, currentFacility, searchQuery)
+      val nearbyFacilitiesPatientsGroup = generateNearbyFacilitiesPatientsGroup(nearbyFacilitiesPatients, currentFacility, searchQuery)
 
       return assignedFacilityPatientsGroup + nearbyFacilitiesPatientsGroup
     }
 
     private fun generateAssignedFacilityPatientsGroup(
         assignedFacilityPatients: List<PatientSearchResult>,
-        currentFacility: Facility
+        currentFacility: Facility,
+        searchQuery: String
     ) = if (assignedFacilityPatients.isNotEmpty()) {
-      listOf(AssignedFacilityHeader(currentFacility.name)) + SearchResult.forSearchResults(assignedFacilityPatients, currentFacility.uuid)
+      listOf(AssignedFacilityHeader(currentFacility.name)) + forSearchResults(assignedFacilityPatients, currentFacility.uuid, searchQuery)
```

Comment:
I think you imported the `SearchResults.Companion.forSearchResults` directly. In this scenario it makes sense to leave it, or else we are missing the context onto why were are doing this.